test/unit/SourceLocator/SourceStubber/PhpStormStubsSourceStubberTest.php

```diff
@@ -359,64 +332,41 @@ public function testInternalFunctions(string $functionName): void
                 'array_diff_uassoc',
                 'array_udiff_uassoc',
                 'array_multisort',
-                'dns_get_record',
                 'extract',
-                'pos',
                 'setcookie',
                 'setrawcookie',
-                'sapi_windows_vt100_support',
+                'stream_context_set_option',
             ], true)
         ) {
             return;
         }
 
-        // Needs fixes in JetBrains/phpstorm-stubs or PHP
-        if (in_array($functionName, ['get_resources', 'sapi_windows_cp_get', 'stream_context_set_option'], true)) {
-            return;
-        }
-
-        // The number of arguments in the signature of this function changed in PHP8
-        if ($originalReflection->getName() !== 'debug_zval_dump' && PHP_VERSION_ID < 80000) {
-            self::assertSame($originalReflection->getNumberOfParameters(), $stubbedReflection->getNumberOfParameters());
-            self::assertSame($originalReflection->getNumberOfRequiredParameters(), $stubbedReflection->getNumberOfRequiredParameters());
-        }
-
         $stubbedReflectionParameters = $stubbedReflection->getParameters();
         foreach ($originalReflection->getParameters() as $parameterNo => $originalReflectionParameter) {
             $parameterName = sprintf('%s.%s', $functionName, $originalReflectionParameter->getName());
 
             $stubbedReflectionParameter = $stubbedReflectionParameters[$parameterNo];
 
-            self::assertSame($originalReflectionParameter->isOptional(), $stubbedReflectionParameter->isOptional(), $parameterName);
+            // Too much errors in JetBrains/phpstorm-stubs
+            // self::assertSame($originalReflectionParameter->isOptional(), $stubbedReflectionParameter->isOptional(), $parameterName);
+
             self::assertSame($originalReflectionParameter->isPassedByReference(), $stubbedReflectionParameter->isPassedByReference(), $parameterName);
             self::assertSame($originalReflectionParameter->canBePassedByValue(), $stubbedReflectionParameter->canBePassedByValue(), $parameterName);
 
-            // Bugs in PHP
-            if (! in_array($parameterName, ['preg_replace_callback.callback', 'header_register_callback.callback'], true)) {
-                self::assertSame($originalReflectionParameter->isCallable(), $stubbedReflectionParameter->isCallable(), $parameterName);
-            }
+            self::assertSame(@$originalReflectionParameter->isCallable(), $stubbedReflectionParameter->isCallable(), $parameterName);
```

Comment:
These `@` should really not be around 🤔

Reply:
It's reported "Deprecated" PHP errors. So:

- I can ignore Deprecated errors in PHPUnit bootstrap
- I can remove the asserts and maybe also remove the methods from BR.

Reply:
I see - let's keep them as they are for now then. A comment when using `@` suffices.

Reply:
In practice, what needs to happen:

 * `@` is considered radioactive :D 
 * if we use it, there must be a clear communication of intent on each occasion we use it

Reply:
https://github.com/Roave/BetterReflection/pull/750/commits/74c288f58837ac88ca8415a790432673351b10c7